boxes/generators/trafficlight.py

```diff
@@ -85,13 +88,13 @@ def topCB(self):
     def frontCB(self):
         self.hole(self.h/2, self.h/2, self.h/2-self.thickness)
 
-    def wall(self, h1, h2, w, edges="ffef", callback=None, move=""):
```

Comment:
Coding style nitpick: No white space around the equal sign for named parameters (in opposite to normal assignment statements).

Reply:
Still one space to much in line 96